### DISCUSSION TITLE
WPS323 Found `%` string formatting

### DISCUSSION THREAD

**Discussion by cidrblock** [ansible/ansible-navigator#704] [Linters, linting, formatting, pre-commit, CI]
The specific line is:

```
logger.error("Subcommand '%s' does not support mode interactive", action)
```

which I thought was still the correct way to format a log message.

**Comment by webknjaz** [comment]
This is a known false-positive, unfortunately. Not sure what to do about it but I think I disabled it exactly because of this in some projects.

There are upstream bugs that seem to be not that straightforward to fix judging by the number of times this issue reoccured in one form or another: https://github.com/wemake-services/wemake-python-styleguide/issues/1489 / https://github.com/wemake-services/wemake-python-styleguide/issues/1879 / https://github.com/wemake-services/wemake-python-styleguide/issues/1304 / https://github.com/wemake-services/wemake-python-styleguide/issues/1796 / https://github.com/wemake-services/wemake-python-styleguide/issues/1329.

Also, this https://github.com/wemake-services/wemake-python-styleguide/issues/1304#issuecomment-607447853 seems to suggest that this check could be ignored if it doesn't work for us. Moreover, since pylint already covers this case, it's probably okay not to care about it that much in flake8.
